### PR TITLE
using FeatureColor in styleFunctions (if possible)

### PR DESCRIPTION
**Pull request by thiloSchlemmer**
update select- and modify-StyleFunction to use the feature-color for styling in
favor of a default color

@alsturm: any further suggestions for styleProperties appreciated

### CODE REVIEW

File: src/modules/map/components/olMap/olStyleUtils.js [hunk 1]
```diff
@@ -212,31 +213,38 @@ export const measureStyleFunction = (feature) => {
...
+			radius: 6,
 			stroke: new Stroke({
-				color: Red_Color,
-				width: 1
+				color: White_Color,
+				width: 2
 			}),
 			fill: new Fill({
-				color: White_Color
+				color: color
```
Comment (taulinger) [inline src/modules/map/components/olMap/olStyleUtils.js:232]: Would suggest adding real test for all style definitions like in `highlight/styleUtils.test.js`
`expect(styles).toBeTruthy()`  =>  `expect(styles).toEqual([style]);`

**Review by taulinger — COMMENTED** [review]
Nice improvement

**Comment by taulinger** [comment]
Coverage of `olStyleUtils` decreased (was 100%)
https://coveralls.io/builds/44848987/source?filename=src%2Fmodules%2Fmap%2Fcomponents%2FolMap%2FolStyleUtils.js

**Review by alsturm — APPROVED** [review]

**Review by taulinger — APPROVED** [review]
Nice!

**Pull request merged**